### ISSUE TITLE
satyr sexuality not obtainible? need help pls

### ISSUE THREAD

**Issue by CapriciousClowns** [Kitteh6660/Corruption-of-Champions-Mod#380]
so far no matter what I do I keep trying everything from drinking satyr wine fucking satyr giving birth to them and interacting with every scene in the game that involves satyr no matter what I never get the perk satyr sexuality how come nobody knows how to get this? its quite difficult is it not even a real perk? or have I been wasting my days trying to unlock a perk that I cant get? :(

**Comment by Darth-Cthoras** [comment]
Looking at CoCED, the save editor, there IS no such thing as Satyr Sexuality. 
https://github.com/tmedwards/CoCEd

**Comment by CapriciousClowns** [comment]
Hi earth cthoras if you look at the mods and releases in one of them kitten said added this to it --ADDITIONS--

Perk Permery! When you ascend, you can make certain perks permanent. Flexibility, Satyr Sexuality, Lustserker, and Ninetails are permable. Marae's Gift perks are permable. It's up to you to find the rest!

**Comment by CapriciousClowns** [comment]
@darh-cthoras

**Comment by Darth-Cthoras** [comment]
Hmmm, digging around in the files, I see there 'is' something called Satyr Sexuality. But it's not listed anywhere in the CoCED itself, and should be gained automatically when turning into a Satyr.

**Comment by Stadler76** [comment]
Found the problem: In `Player.satyrScore()` the wrong lowerBody-type is being checked for.
Here's the diff:
```diff
diff --git a/classes/classes/Player.as b/classes/classes/Player.as
index 9dc6ba5..08ab31d 100644
--- a/classes/classes/Player.as
+++ b/classes/classes/Player.as
@@ -1640,7 +1640,7 @@ use namespace kGAMECLASS;
 		public function satyrScore():Number
 		{
 			var satyrCounter:Number = 0;
-			if (lowerBody == LOWER_BODY_TYPE_HOOFED)
+			if (lowerBody == LOWER_BODY_TYPE_CLOVEN_HOOFED)
 				satyrCounter++;
 			if (tailType == TAIL_TYPE_GOAT)
 				satyrCounter++;
```

**Comment by CapriciousClowns** [comment]
That's weird I did change my lower body to cloven hoofed but it would never give me the satyr sexuality and you know I tried a lot of things to get it 100 days wasted fucking Satyr drinking satyr wine and trying everything to get it also you guys might play this game for the sexual fun and thrill but I really like to play this game because of the awesome story and fighting sequences it be cool if the game had more fighting thrill in it which it already does but wouldn't hurt to do some more but yeah I like this game because it sexy fun and entertaining game c;

**Comment by CapriciousClowns** [comment]
Oh yeah also the character thing where you can read up on yourself I was a mutant but sooner or later it changed and called me a cowgirl and I was so fuckinge confused lol I was no where from being called a cow girl I was a full on mutant with each body part different I wonder if that migjt be worth adding into the issue?

**Comment by Darth-Cthoras** [comment]
Oh, good job, Stadler. Thanks!

**Comment by Kitteh6660** [comment]
From the look, it should now be resolved already and will come in 1.4.
